gui/templates/system/bootenv_keep.html

```diff
@@ -0,0 +1,6 @@
+{% extends "freeadmin/generic_form.html" %}
+
+{% block form %}
+<tr><td colspan="2"><span>Do You Want To Keep BootEnv</span></td></tr>
```

Comment:
Please use translation here, also do not use upper case for every word.

{% trans "Do you want...." %}